API.md

```diff
@@ -0,0 +1,48 @@
+# AmperkaTDS API
+
+## `class AmperkaTDS`
+
+Create an object of type `AmperkaTDS` to communicate with a particular [TDS-sensor](https://amperka.ru/product/troyka-tds-sensor).
+
+### `AmperkaTDS(uint8_t pinS)`
+
+Constructs a new `AmperkaTDS` object. The argument `pinS` the analog output signal. It is marked `S` on the TDS board. Use any analog GPIO pin of the controller.
+
+### `void begin()`
+
+Initializes the given interface, prepares the board for communication.
+Call this method before interfacing with `AmperkaTDS`. For example, in your `setup()`.
+
+### `void read()`
```

Comment:
In this case, we can make the read method private, as it will only read the analog value from the sensor. And the calculation of the TDS and the EC can be done already in the methods getTDS and getEC.

Reply:
:ok_hand: